packages/wallet/ui/config-overrides/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "wallet-ui-overrides",
```

Comment:
Please always prefix package names to prevent confusion via npm squatters.

```suggestion
  "name": "@agoric/wallet-ui-overrides",
```

Reply:
Done

Reply:
Do we even need a package name? I don't see this used anywhere

Reply:
This is a hack to let the override script be in CommonJS, since it seems to be necessary, while `@agoric/wallet-ui` is `"type": "module"`.

Reply:
You don't need to give a name. Also why not use the `.cjs` extension?

Reply:
Removed `name`. Discussed offline: `.cjs` wasn't working